docs/html/PULL__REQUEST__TEMPLATE_8md.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
Se o markdown do PR for apagado, naturalmente, esse arquivo também será apagado.